packages/p2p/src/components/app.jsx

```diff
@@ -136,7 +136,13 @@ class App extends Component {
                 }}
             >
                 <main className={classNames('deriv-p2p', className)}>
-                    <Tabs onTabItemClick={this.handleTabClick} active_index={active_index} top>
+                    <Tabs
+                        onTabItemClick={this.handleTabClick}
+                        active_index={active_index}
+                        className='deriv-p2p'
```

Comment:
we shouldn't use brand name here, p2p is imported in binary too, so it should be brand-agnostic.

Reply:
alright. Thank you 👍

Reply:
should I also change the classname in line 138?